main.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2021 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/mendersoftware/go-lib-micro/config"
+	log "github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+
+	dconfig "github.com/mendersoftware/azure-iot-manager/config"
+	"github.com/mendersoftware/azure-iot-manager/server"
+)
+
+func main() {
+	doMain(os.Args)
+}
+
+func doMain(args []string) {
+	var configPath string
+
+	app := &cli.App{
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name: "config",
+				Usage: "Configuration `FILE`. " +
+					"Supports JSON, TOML, YAML and HCL " +
+					"formatted configs.",
+				Value:       "/etc/azure-iot-manager/config.yaml",
+				Destination: &configPath,
+			},
+		},
+		Commands: []cli.Command{
+			{
+				Name:   "server",
+				Usage:  "Run the HTTP API server",
+				Action: cmdServer,
+				Flags: []cli.Flag{
+					&cli.BoolFlag{
+						Name:  "automigrate",
+						Usage: "Run database migrations before starting.",
+					},
+				},
+			},
+			{
+				Name:   "migrate",
+				Usage:  "Run the migrations",
+				Action: cmdMigrate,
+			},
+		},
+	}
+	app.Usage = "Device Connect"
+	app.Version = "1.0.0"
```

Comment:
We should stop hardcoding the version to `1.0.0`. This is only adding confusion. IMO we should just omit this line.

Reply:
I have no idea, nad no problem with omitting this line, the problem is, we are doing it for others repos. how about keeping it "for now" and discussing tomorrow during BET meeting? (the discussion about versioning of backend services is long over due), makes sense?

Reply:
We discussed it over PRs that we had some months ago. My suggestion is to remove it from here to stop propagating it to new repos.

Reply:
removed.
yes, and I asked back then if we can discuss removing versioning of backend services. :)